client/src/client.rs

```diff
@@ -247,6 +247,15 @@ pub struct ResultSet<T> {
     client_cursor: usize,
 }
 
+impl<T> ResultSet<T> {
+    /// Get the length of the batch returned by Iroha.
+    ///
+    /// This is controlled by `fetch_size` parameter of the query.
+    pub fn batch_len(&self) -> usize {
+        self.iter.len()
+    }
+}
+
```

Comment:
```suggestion
```

I don't think we need this. The one place you use it, you could have just collected into a vec

Reply:
No, I can't, because it will collect all batches implicitly